src/tags.h

```diff
@@ -315,26 +316,25 @@ struct Data {
   using Value_t = detail::TagImpl<detail::DataDef::Value>;
   using Variance_t = detail::TagImpl<detail::DataDef::Variance>;
   using StdDev_t = detail::TagImpl<detail::DataDef::StdDev>;
-  using Int_t = detail::TagImpl<detail::DataDef::Int>;
-  using DimensionSize_t = detail::TagImpl<detail::DataDef::DimensionSize>;
-  using String_t = detail::TagImpl<detail::DataDef::String>;
+  // TODO Int and String is deprecated and should be removed, it is currently
+  // only required to maintain tests using MDZipView before it is properly
+  // refactored for multi-name support.
+  using DeprecatedInt_t = detail::TagImpl<detail::DataDef::Int>;
```

Comment:
It's not clear to me why `Data::Int` was necessary is this simply to get a Tag with an underlying type of int given that it couldn't be derived from the data itself?

Reply:
Previously the type was defined by the tag, see we needed it as a separate tag. Now it is not necessary any more, except that we cannot actually remove it until `MDZipView` is refactored.

Reply:
Ok. That's pretty much in line with what I understand now.